ogles_gpgpu/common/proc/mesh.cpp

```diff
@@ -0,0 +1,129 @@
+//
+// ogles_gpgpu project - GPGPU for mobile devices and embedded systems using OpenGL ES 2.0
+//
+// See LICENSE file in project repository root for the license.
+//
+
+// Copyright (c) 2017-2018, David Hirvonen (this file)
+
+#include "mesh.h"
+
+BEGIN_OGLES_GPGPU
+
+// clang-format off
+const char * MeshShaderProc::vshaderMeshSrc = OG_TO_STR
+(
+ attribute vec4 aPos;
+ attribute vec2 aTexCoord;
+ varying vec2 vTexCoord;
+ uniform mat4 transformMatrix;
+ void main()
+ {
+    gl_Position = transformMatrix * vec4(aPos.xyz, 1.0);
+    vTexCoord = aTexCoord;
+ }
+ );
+// clang-format on
+
+// clang-format off
+const char * MeshShaderProc::fshaderMeshSrc =
+#if defined(OGLES_GPGPU_OPENGLES)
+ OG_TO_STR(precision mediump float;)
+#endif
+ OG_TO_STR(
+ varying vec2 vTexCoord;
+ uniform sampler2D uInputTex;
+ void main()
+ {
+    gl_FragColor = vec4(texture2D(uInputTex, vTexCoord).rgba);
+ }
+);
+// clang-format on
+
+MeshShaderProc::MeshShaderProc(const VertexBuffer& vertices, const CoordBuffer& coords)
+    : texUnit(1)
+    , texTarget(GL_TEXTURE_2D)
+    , vertices(vertices)
+    , coords(coords)
+{
+    memset(MVP.data, 0, sizeof(MVP.data));
```

Comment:
On Linux:
```
error: ‘memset’ was not declared in this scope
     memset(MVP.data, 0, sizeof(MVP.data));
```
I think `#include <string.h>` will fix it:
* https://linux.die.net/man/3/memset

Reply:
Replaced with for loop initialization.